src/current/v25.3/citext.md

```diff
@@ -0,0 +1,98 @@
+---
+title: CITEXT
+summary: The CITEXT data type stores case-insensitive text values.
+toc: true
+docs_area: reference.sql
+---
+
+The `CITEXT` [data type]({% link {{ page.version.version }}/data-types.md %}) represents a case-insensitive string. Like `STRING` values, `CITEXT` values preserve their casing when stored and retrieved. Unlike `STRING` values, comparisons between `CITEXT` values are case-insensitive for all Unicode characters that have a defined uppercase/lowercase mapping (e.g., `'É' = 'é'`).
```

Comment:
"Unicode characters" could be a link to e.g. https://en.wikipedia.org/wiki/List_of_Unicode_characters